queries/fragments/collectionDetails.graphql

```diff
@@ -0,0 +1,4 @@
+fragment collectionDetails on CollectionEntity {
```

Comment:
formatting ser 👀

Reply:
wow.. no idea what happened there. looked totally fine in VSCode 😅